docs/src/python/atleast_utils.rst

```diff
@@ -0,0 +1,15 @@
+.. _utils:
+
+At least Utils
```

Comment:
Don't need a separate page in the docs for this, I would just put them in the `ops.rst` list.